Update the W/X quickstart articles when the Dev Kit GA (#4902)
OfficeDev/office-js-docs-pr: docs/quickstarts/excel-quickstart-jquery.md

# Build an Excel task pane add-in

> In this article, you'll walk through the process of building an Excel task pane add-in. You'll use either the Office Add-ins Development Kit or the Yeoman generator to create your Office Add-in. Select the tab for the one you'd like to use and then follow the instructions to create your add-in and test it locally. If you'd like to use Visual Studio Code, we recommend the Office Add-ins Development Kit.

Reviewer: ```suggestion
In this article, you'll walk through the process of building an Excel task pane add-in. You'll use either the Office Add-ins Development Kit or the Yeoman generator to create your Office Add-in. Select the tab for the one you'd like to use and then follow the instructions to create your add-in and test it locally. If you'd like to create the add-in project within Visual Studio Code, we recommend the Office Add-ins Development Kit.
```

# Build an Excel task pane add-in

In this article, you'll walk through the process of building an Excel task pane add-in. You'll use either the Office Add-ins Development Kit or the Yeoman generator to create your Office Add-in. Select the tab for the one you'd like to use and then follow the instructions to create your add-in and test it locally. If you'd like to use Visual Studio Code, we recommend the Office Add-ins Development Kit.

# [Office Add-ins Development Kit](#tab/devkit)
## Prerequisites

[!include[Dev_kit prerequisites](../includes/dev-kit-prerequisites.md)]

## Create the add-in project

Click the following button to create an add-in project using the Office Add-ins Development Kit for Visual Studio Code. A page that contains the project decription will open in Visual Studio Code.

> [!div class="nextstepaction"]
> [Create an add-in in Visual Studio Code](vscode://msoffice.microsoft-office-add-in-debugger/open-specific-sample?sample-id=Excel-HelloWorld-TaskPane-JS)

In the prompted page, select **Create** to create the add-in project. In the **Workspace folder** dialog that opens, select the folder where you want to create the project. 

> <img src="../images/office-add-ins-development-kit-samplepage-excelquickstart.png" alt="Screenshot shows the sample page in Visual Studio Code"/>

Reviewer 2: Favor the [Learn image markdown syntax](https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?branch=main#custom-learn-markdown-for-standard-conceptual-images-recommended)
```suggestion
:::image type="content" source="../images/office-add-ins-development-kit-samplepage-excelquickstart.png" alt-text="The Development Kit sample gallery in Visual Studio Code":::
```

The Office Add-ins Development Kit will create the project. It will then open the project in a *second* Visual Studio Code window. Close the original Visual Studio Code window.

> [!NOTE]
> You will receive prompts to install the Office Add-ins Development Kit extension if you haven't. You can also [install the Office Add-ins Development Kit manually](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/development-kit-overview?tabs=vscode).

## Explore the project

[!include[Devkit_project_components_taskpane](../includes/devkit-project-components-taskpane.md)]

## Try it out

[!include[Dev_kit_start_debugging](../includes/dev-kit-start-debugging.md)]

## Stop testing your Office Add-in

[!include[Dev_kit_stop_debugging](../includes/dev-kit-stop-debugging.md)]

## Troubleshooting

[!include[Dev_kit_troubleshooting](../includes/dev-kit-troubleshooting.md)]

> 
> 

Reviewer 2: ```suggestion

```

# Build an Excel task pane add-in

> In this article, you'll walk through the process of building an Excel task pane add-in.
> 
> You can create an Office Add-in by using the Office Add-ins Development Kit or the Yeoman generator in Visual Studio Code. Select the tab for the one you'd like to use and then follow the instructions to create your add-in and test it locally.

Reviewer 2: ```suggestion
In this article, you'll walk through the process of building an Excel task pane add-in. You'll use either the Office Add-ins Development Kit or the Yeoman generator to create your Office Add-in. Select the tab for the one you'd like to use and then follow the instructions to create your add-in and test it locally. If you'd like to use Visual Studio Code, we recommend the Office Add-ins Development Kit.
```

# [Office Add-ins Development Kit](#tab/devkit)

> ## Prerequisites

Author: The prerequisite section is same as the one section in (docs/develop/development-kit-overview.md)

- Download and install [Visual Studio Code](https://code.visualstudio.com/).
- Node.js (the latest LTS version). Visit the [Node.js site](https://nodejs.org/) to download and install the right version for your operating system. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
- Office connected to a Microsoft 365 subscription. You might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program), see [FAQ](/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-) for details. Alternatively, you can [sign up for a 1-month free trial](https://www.microsoft.com/microsoft-365/try?rtc=1) or [purchase a Microsoft 365 plan](https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).

## Create the add-in project

Click the following button to create an add-in project using the Office Add-ins Development Kit for Visual Studio Code. A page that contains the project decription will open in Visual Studio Code.

> <button type="button">[Create an add-in in Visual Studio Code](vscode://msoffice.microsoft-office-add-in-debugger/open-specific-sample?sample-id=Excel-HelloWorld-TaskPane-JS)</button>

Author: I imagine a clickable button to link to the sample page in VSC. Can we achieve that in MS Learn site?

Reviewer 2: I think this is the best way to do that: https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?branch=main#next-step-action. I've added the markdown as a suggestion.

> <button type="button">[Create an add-in in Visual Studio Code](vscode://msoffice.microsoft-office-add-in-debugger/open-specific-sample?sample-id=Excel-HelloWorld-TaskPane-JS)</button>

Reviewer 2: ```suggestion
> [!div class="nextstepaction"]
> [Create an add-in in Visual Studio Code](vscode://msoffice.microsoft-office-add-in-debugger/open-specific-sample?sample-id=Excel-HelloWorld-TaskPane-JS)
```

<img src="../images/office-add-ins-development-kit-samplepage-excelquickstart.png" alt="Screenshot shows the sample page in Visual Studio Code"/>

> In the prompted page, select **Create** to create the add-in project. In the **Workspace folder** dialog that opens, select the folder where you want to create the project. 

Reviewer 2: I'd move the screenshot so it follow the "Create" instruction. In the screenshot, surround the Create button with a red box to highlight where it is in the UI.

> Office Add-ins Development Kit will create the project. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.

Reviewer 2: I think limiting the italics to the keyword will make it more obvious for readers.
```suggestion
The Office Add-ins Development Kit will create the project. It will then open the project in a *second* Visual Studio Code window. Close the original Visual Studio Code window.
```

> [!NOTE]

> > You will receive prompts to install the Office Add-ins Development Kit extension if you haven't. You can also [install the Office Add-ins Development Kit manually](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/development-kit-overview?tabs=vscode).

Reviewer 2: ```suggestion
> You'll be prompted to install the Office Add-ins Development Kit extension if don't already have it. If you need to install the Office Add-ins Development Kit manually, follow [these steps](../development-kit-overview.md?tabs=vscode).
```

> ## Explore the project

Author: The "Explore the project"section is most similar as the one in the yeoman option.

Reviewer 2: Are they the same? If so, we should make an [include file](https://review.learn.microsoft.com/en-us/help/platform/includes-best-practices?branch=main).

The add-in project that you've created with the Office Add-ins Development Kit contains sample code for a basic task pane add-in. If you'd like to explore the components of your add-in project, open the project in your code editor and review the files listed below. When you're ready to try out your add-in, proceed to the next section.

1. The ./manifest.xml or manifest.json file in the root directory of the project defines the settings and capabilities of the add-in.
1. The ./src/taskpane/taskpane.html file contains the HTML markup for the task pane.
1. The ./src/taskpane/taskpane.css file contains the CSS that's applied to content in the task pane.
1. The ./src/taskpane/taskpane.js file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office client application.

> ## Try it out

Author: "Try it out", "Stop testing", "Troubleshooting" are same as the ones in (docs/develop/development-kit-overview.md)

Reviewer 2: Same suggestion for an include file. This prevents content from being different in two places if we miss something in an update.

Reviewer 2: Is there a reason we didn't include the headers in the include files? I think you could make this whole thing one include file.

Author: For the "start debugging" one, there is other content after the header in the dev kit overview doc, so the header is not in the include file. Others are included.
